nestkernel/nest_time.h

```diff
@@ -537,6 +562,9 @@ class Time
   {
     return ld_round( ms * Range::STEPS_PER_MS );
   }
+
+private:
+  unsigned long steps_;
```

Comment:
Time can be negative, so `steps_` must be `long`, not `unsigned long`. This, unfortunately, makes it difficult to mark `steps_` as invalid by assigning a special value.